ddtrace/settings/config.py

```diff
@@ -193,6 +193,18 @@ def __init__(self):
             "DD_TRACE_PROPAGATION_STYLE_INJECT", default=PROPAGATION_STYLE_DATADOG
         )
 
+        # Datadog tracer tags propagation
+        x_datadog_tags_max_length = int(os.getenv("DD_TRACE_X_DATADOG_TAGS_MAX_LENGTH", default=128))
```

Comment:
Based on the spec, should the default here be **512** instead of **128**?

The file in `ddtrace/internal/_tagset.pyx` also mentions that the default is **512**.

Reply:
Good catch!